### DISCUSSION TITLE
Printing particular item

### DISCUSSION THREAD

**Discussion by punfil** [benjaminjonard/koillection#739] [Ideas]
Hi!
I am new user to Koillection. I really enjoy the software, as it's very simple and does it's job :)
However, for my usage I would need to print a particular item into a small label that I can put on a shelf with the item.
Is there any chance to add such feature?
Thanks

**Comment by mikeymakesit** [comment]
I have very similar desire.  It would be great when viewing an item or collection to be shown (or toggle on/off) a QR code which is a link to that page/view.
